pkg/generate/code/set_resource.go

```diff
@@ -962,6 +962,8 @@ func SetResourceIdentifiers(
 		case r.Ops.ReadMany != nil:
 			// If single lookups can only be done using ReadMany
 			op = r.Ops.ReadMany
+		default:
+			return ""
```

Comment:
Not sure why is this needed, it could also break other functionalities

Reply:
`If op == nil` on line 957 is true, then we get into this `switch` statement. However, if both cases `r.Ops.GetAttributes` and `r.Ops.ReadMany` are `nil`, then `op` remains null. and then the code generator crashes on line 969 when it dereferences `op` as `op.InputRef.Shape`.

This crash happened when I was testing `networkAclEntry` which does not have `read` operation and `op` remained `nil`. 

So fixing this already existing bug.